core/deployment/pom.xml

```diff
@@ -133,6 +133,18 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemProperties>
```

Comment:
This is deprecated: https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#systemProperties

> Deprecated. Use systemPropertyVariables instead.

Reply:
Ah, good catch!

Reply:
Probably best to just change this now and we can deal with the others in another PR